binding.gyp

```diff
@@ -1,21 +1,31 @@
 {
-  "targets": [
-    { "target_name": "" }
-  ],
+  "targets": [{
+    "target_name": ""
+  }],
   "conditions": [
     ['OS=="mac"', {
       "targets": [{
-        "target_name": "fse",
-        "sources": ["fsevents.cc"],
-        "xcode_settings": {
-          "OTHER_LDFLAGS": [
-            "-framework CoreFoundation -framework CoreServices"
+          "target_name": "<(module_name)",
+          "sources": ["fsevents.cc"],
+          "xcode_settings": {
+            "OTHER_LDFLAGS": [
+              "-framework CoreFoundation -framework CoreServices"
+            ]
+          },
+          "include_dirs": [
+            "<!(node -e \"require('nan')\")"
```

Comment:
Can you undo the whitespace and stylistic changes in this block?  It makes the diff noisier than it needs to be.  Thanks.